language_service/src/completion.rs

```diff
@@ -289,7 +289,7 @@ impl CompletionListBuilder {
     }
 
     fn push_attributes(&mut self) {
-        static ATTRIBUTES: [&str; 2] = ["@EntryPoint()", "@Config()"];
+        static ATTRIBUTES: [&str; 3] = ["@EntryPoint()", "@Config()", "@Measurement()"];
```

Comment:
Heads up, this code is changing in my PR. When you hit the merge conflict, I suggest just reverting this and searching again for the attribute list instead of trying to resolve it as a merge conflict...